pylicenses/providers/piplocal.py

```diff
@@ -10,7 +10,8 @@
 
 
 class PipProvider(PackageProvider):
-    def get_package_metadata(self, pkg):
+    @staticmethod
```

Comment:
should not be a staticmethod (it is called on the class instance)

Reply:
fixed